Makefile

```diff
@@ -31,9 +29,7 @@ release:
 		--workdir /cilium \
 		--volume `pwd`:/cilium docker.io/library/golang:1.19.3-alpine3.16 \
 		sh -c "apk add --no-cache make git && \
```

Comment:
I think you need to install setpriv first. The builtin one provided by busybox does not have the necessary flags.

Reply:
Fixed, thanks.